tooling/minimize/src/program.rs

```diff
@@ -126,6 +130,8 @@ pub struct FnCtxt<'cx, 'tcx> {
 
     pub locals: Map<LocalName, Type>,
     pub blocks: Map<BbName, BasicBlock>,
+
+    pub locals_stable: Vec<smir::LocalDecl>,
```

Comment:
This seems to have a role quite different from `locals`, it's really part of the body. Could you move it next to the body, and add a doc comment explaining its purpose?